pagopa/io-app
# chore(IT Wallet): [SIW-1399] Remove cta from presentation credential footer

## Short description
This PR remove CTA from presentation credential footer screen.

## List of changes proposed in this pull request
- Update ItwPresentationCredentialScreen

## How to test
Launch the app, get a credential, and check if the Remove from wallet CTA has disappeared in the footer section of the details screen
